instrumentation/logback-classic-1.2/build.gradle

```diff
@@ -0,0 +1,18 @@
+jar {
+    manifest { attributes 'Implementation-Title': 'com.newrelic.instrumentation.logback-classic-1.2' }
+}
+
+dependencies {
+    implementation(project(":agent-bridge"))
+    implementation("ch.qos.logback:logback-classic:1.2.6")
+}
+
+verifyInstrumentation {
+    passes("ch.qos.logback:logback-classic:[1.0,)")
+    excludeRegex 'ch.qos.logback:logback-classic:1.3.*'
```

Comment:
This regex exclusion is too broad as it will prevent the instrumentation from applying to any legitimate 1.3.x versions that are released.

Does the instrumentation actually fail to verify against the `1.3.0-alphaX` artifact versions? Or are you simply excluding them because they're `alpha` releases? If it's the latter, then the regex should be updated to only capture artifacts with `alpha` in their names. If this instrumentation does begin failing against 1.3.X versions, then the excludeRegex should be removed and the `passes` range should be changed to `passesOnly("ch.qos.logback:logback-classic:[1.0.0,1.3.0-alpha0)")`. This will ensure that all artifacts from 1.0.0 up to, but not including 1.3.0-alpha0, pass verification while verifying that all other artifacts outside the range fail against the instrumentation.

Reply:
Ok, I ran this against the instrumentation verifier and the following succeeds. That block should be changed to the following as the instrumentation applies to versions 0.9.3+ (though 0.9.6 fails for some reason, hence the exclude):

```
verifyInstrumentation {
    passesOnly("ch.qos.logback:logback-classic:[0.9.3,)")
    excludeRegex '.*(alpha|groovyless).*'
    excludeRegex 'ch.qos.logback:logback-classic:0.9.6'
}
```